### PR TITLE
acm: add acm ci cluster roles

### PR DESCRIPTION
**Pull request by ray-harris**

### CODE REVIEW

File: cluster/ci/config/roles.yaml [hunk 1]
```diff
@@ -868,3 +868,52 @@ items:
...
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: admin
+    namespace: open-cluster-management
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: ocm-admins
+  - kind: Group
+    name: ci-admins
```
Comment (hongkailiu) [inline cluster/ci/config/roles.yaml:895]: No need to add ci-admins ... they are admins already.

File: cluster/ci/config/roles.yaml [hunk 2]
```diff
@@ -868,3 +868,52 @@ items:
...
+    apiGroup: rbac.authorization.k8s.io
+  - kind: ServiceAccount
+    name: ci-operator
```
Comment (hongkailiu) [inline cluster/ci/config/roles.yaml:898]: Why you need this?

File: cluster/ci/config/roles.yaml (hunk 1 above)
Comment (ray-harris) [reply cluster/ci/config/roles.yaml:895]: I'll take that one out.

File: cluster/ci/config/roles.yaml (hunk 2 above)
Comment (ray-harris) [reply cluster/ci/config/roles.yaml:898]: When this job ran: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/branch-ci-open-cluster-management-image-builder-master-images/1

It failed with this error:
```error: could not run post step : could not ensure target imagestream: imagestreams.image.openshift.io "builder" is forbidden: User "system:serviceaccount:ci:ci-operator" cannot get imagestreams.image.openshift.io in the namespace "open-cluster-management": no RBAC policy matched```

After running this command:
```oc adm policy add-role-to-user admin system:serviceaccount:ci:ci-operator -n open-cluster-management```

The job run successfully and pushed the image. There may be more restrictive permissions that will work or some other way to get around the error, but that's what worked.

Comment (hongkailiu) [reply cluster/ci/config/roles.yaml:898]: You do not need to be project admin to push.
See `image-tagger-ci-operator` for example.

Comment (ray-harris) [reply cluster/ci/config/roles.yaml]: Ok. Pushing commit to fix.

**Pull request merged**